pkgs/top-level/python-packages.nix

```diff
@@ -7217,6 +7217,7 @@ in {
       description = "Push and pull from a Git server using Mercurial";
       homepage = http://hg-git.github.com/;
       maintainers = with maintainers; [ koral ];
+      license = stdenv.lib.licenses.gpl2;
```

Comment:
[Source: COPYING in source repo](https://bitbucket.org/durin42/hg-git/src/70c9dfdb0bc1091b436f2ac6b5458cc10af7a52e/COPYING)